src/prefs.xhtml

```diff
@@ -0,0 +1,26 @@
+   <vbox>
+        <groupbox>
+            <caption label="OCR parameters"/>
+            <label value="Full location of the tesseract executable (when empty, some standard locations are searched for it):"/>
+            <html:input type="text" id="pref-zoteroocr-ocrPath-value" flex="1" preference="extensions.zotero.zoteroocr.ocrPath"/>
+            <label value="Full location of the pdftoppm executable:"/>
+            <html:input type="text" id="pref-zoteroocr-pdftoppmPath-value" flex="1" preference="extensions.zotero.zoteroocr.pdftoppmPath"/>
+            <hbox>
+                <label value="Choose a language/script you want to use for recognition (default is eng):"/>
+                <html:input type="text" id="pref-zoteroocr-language-value" preference="extensions.zotero.zoteroocr.language" width="100"/>
+            </hbox>
+        </groupbox>
+        <groupbox>
+            <caption label="Output options"/>
+            <checkbox preference="extensions.zotero.zoteroocr.outputNote" label="Save output as a note"/>
+            <checkbox id="checkbox-zoteroocr-output-pdf" preference="extensions.zotero.zoteroocr.outputPDF" label="Save output as a PDF with text layer" oncommand="Zotero.OCR.updatePDFOverwritePref()"/>
```

Comment:
In Zotero 6 the OCR was triggered from the context menu. Using a checkbox now might be sufficient for a first test, but is not the right solution for Zotero 7.